dropwizard-request-logging/src/main/java/io/dropwizard/request/logging/async/AsyncAccessEventAppenderFactory.java

```diff
@@ -16,11 +16,6 @@
      */
     @Override
     public AsyncAppenderBase<IAccessEvent> build() {
-        return new AsyncAppenderBase<IAccessEvent>() {
-            @Override
-            protected void preprocess(IAccessEvent event) {
-                event.prepareForDeferredProcessing();
-            }
-        };
+        return new AsyncAppenderBase<IAccessEvent>();
```

Comment:
Could you please also update the Javadoc for this method? In looks a lit bit wrong for me.

* It should link to `AsyncAppenderBase` rather than to `AsyncAppenderFactory` (not-related issue to this change)
* I think we should remove a claim about deferred processing.